csm/core/services/rgw/s3/users.py

```diff
@@ -43,3 +43,17 @@ async def create_user(self, **user_body):
         if isinstance(plugin_response, RgwError):
             self._handle_error(plugin_response)
         return plugin_response
+
+    @Log.trace_method(Log.DEBUG, exclude_args=['access_key', 'secret_key'])
+    async def add_access_key(self, **add_access_key_body):
```

Comment:
please rename: add_access_key -> create_key

Reply:
renamed to create_key